IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_Purchase/DocPropertySet.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_Purchase" UniqueId="6bf8a00d-dab0-4a42-8d32-461e590b9854" ApplicableType="IfcElement" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="PurchasePrice_0ScQezQyP2duRs0dFE3Jzw" />
+		<DocProperty xsi:nil="true" href="Supplier_3wkQvqvdXEkRfl5wppCLK_" />
```

Comment:
I find this PSet a bit troubling as the situation it is addressing is far more complex than just a price and supplier.

Consider researching and using IfcProjectOrder to properly portray and and exchange this information?

Reply:
How about considering actually modelling cost items?